main.ts

```diff
@@ -0,0 +1,53 @@
+const { readFile, writeFile } = require("fs");
+
+// Creates clojure
+// Curried version:
+// calculate number of particular symbol in string
+const numOfSymbols = (symbol: string) => (str: string) => Array
+    // array from a given string
+    .from(str)
+    // if element in array equals given symbol - increase number
+    .reduce((acc, curr) => curr.toLowerCase() === symbol
+        ? (acc + 1)
+        : acc,
+        0
+    );
+
+
+// read file, calculate number of appearences of particular symbol,
+// create new file, put the number there
+const formReadRes = (readFileName: string) => (writeFileName: string) => (symbolToCalc: string) => {
```

Comment:
does "formReadRes" give any hints about what it does? For me - it doesn't. Why not call it smth like `writeNumSymbolsInFileIntoFile()`, this is long, but somewhat explaining what it does.

Also, keep in mind to not overuse currying just everywhere. It's a nice feature, but outside of purely functional languages it will be treated mostly as obnoxious by other programmers, unless it' done with some use case in mind (like the ones I showed you).